packages/core/src/modules/dids/__tests__/peer-did.test.ts

```diff
@@ -11,15 +11,15 @@ import { convertPublicKeyToX25519, getEd25519VerificationMethod } from '../domai
 import { getX25519VerificationMethod } from '../domain/key-type/x25519'
 import { DidKey } from '../methods/key'
 import { DidPeer, PeerDidNumAlgo } from '../methods/peer/DidPeer'
-import { DidDocumentRecord, DidDocumentRepository } from '../repository'
```

Comment:
@jakubkoci there are a few changes to how you should retrieve and store the did document